File: FTIndicatorTest/FTIndicatorTest/ViewController.m
//
//  ViewController.m
//  FTIndicatorTest
//
//  Created by liufengting on 2016/10/24.
//  Copyright © 2016年 LiuFengting. All rights reserved.
//

#import "ViewController.h"
#import <FTIndicator.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

}

- (IBAction)buttonTapped:(UIButton *)sender
{
    [FTIndicator showProgressWithmessage:@"Some Messages." userInteractionEnable:NO];
}

@end

File: FTImageViewer/FTImageViewer/ViewController.swift
//
//  ViewController.swift
//  FTImageViewer
//
//  Created by liufengting on 2016/10/24.
//  Copyright © 2016年 LiuFengting. All rights reserved.
//

import UIKit
import FTIndicator

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

    
    }

    @IBAction func buttonTapped(_ sender: UIButton) {
        
        FTIndicator.showProgressWithmessage("Some messages.", userInteractionEnable: false)
        
    }
}

